<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title>Clostridium immunis:   A human gut commensal with therapeutic immunomodulatory effects</title>






<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->
<link href='https://fonts.googleapis.com/css?family=Palatino' rel='stylesheet'>
<link href='https://fonts.googleapis.com/css?family=Rasa' rel='stylesheet'>

<!--
Here are the required style attributes for css to make this poster work :)
-->
<style>
@page {
size: 46in 32in;
margin: 0;
padding: 0;
background-color: #ffffff;
}
body {
margin: 0px;
padding: 0px;
width: 46in;
height: 32in;
text-align: justify;
font-size: 40px;
background-color: #FFFFFF;
color: #000000;
}
/* RMarkdown Class Styles */
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
margin: auto;
margin-top: 1em;
margin-bottom: 1em;
/* align-items: center; */
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
code {
font-size: 25pt;
font-family: monospace;
background-color: #00808024;
color: #0b4545;
padding: 1.2mm;
line-height: 1;
border-radius: 2mm;
}
caption {
margin-bottom: 5px;
font-size: 20pt;
font-style: italic;
}

tbody tr:nth-child(odd) {
    background-color: #0b454520;
}
.table>thead>tr>th, .table>tbody>tr>th, .table>tfoot>tr>th, .table>thead>tr>td, .table>tbody>tr>td, .table>tfoot>tr>td{
  border-spacing: 0;
  font-size: 40%;
  border-style: none;
  padding-top: 15px;
  padding-bottom: 15px;
  padding-right: 1em;
  padding-left: 1em;
  line-height: 1em;
}
table {
  margin: auto;
}
th {
  padding-left: 5mm;
  padding-right: 5mm;
}
.caption {
font-size: 20pt;
font-style: italic;
padding-top: 0;
}
.references {
font-size: 20px;
line-height: 90%;
}
/* Create three unequal columns that floats next to each other */
.column {
float: left;
padding: 0px;
}
.outer {
width: 46in;
height: 32in;
-webkit-column-count: 2; /* Chrome, Safari, Opera */
-moz-column-count: 2; /* Firefox */
column-count: 2;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
-webkit-column-rule-width: 50%;
-moz-column-rule-width: 50%;
column-rule-width: 50%;
-webkit-column-rule-style: solid;
-moz-column-rule-style: solid;
column-rule-style: solid;
-webkit-column-rule-color: black;
-moz-column-rule-color: black;
column-rule-color: black;
/* we need a gap of exactly or more than the main-width between column 1 and column 2 = >33.33% */
-webkit-column-gap: 34%;
background-color: #ffffff;
font-family: Rasa;
color: #000000;
}
span.citation {
  color: #008080;
  font-weight: bold;
}
a {
text-decoration: none;
color: #008080;
}
#title {
font-size: 80pt;
text-align: left;
margin: 0;
line-height: 90%;
border-bottom: 0;
}
#author {
color: #0b4545;
margin: 0;
line-height: 90%;
font-size: 24pt;
}
#affiliation {
padding-top: 1em;
color: #00000060;
font-style: italic;
font-size: 25px;
margin: 0;
}
sup {
color: #cc0000;
}
.affiliation sup {
font-size: 20px;
}
.author sup {
font-size: 30px;
}
.author_extra {
color: #008080;
margin: 0;
line-height: 85%;
font-size: 35px;
}
.outer h1 {
text-align: center;
margin-top: 0.5in;
margin-bottom: 0.5in;
}
.outer h2 {
text-align: center;
}
.outer p {
color: #000000;
}
.outer ol {
text-align: left;
}
.main {
width: calc(46in / 3);
height: 32in;
position: absolute;
margin-left: calc(46in / 3);
background-color: #0b4545;
color: #FFFFFF90;
font-family: Palatino;
background-image: linear-gradient(#0b4545 50%, #008080);
}
.main strong {
color: #FFFFFF;
}
img.main-img-left {
width: 20%;
left: 0.5in;
bottom: 0.2in;
position: absolute;
}
img.main-img-right {
width: 18%;
right: 0.5in;
bottom: 0.2in;
position: absolute;
}
img.main-img-mid {
width: 20%;
right: 40%;
bottom: 20%;
position: absolute;
}
.main p {
padding-top: 20%;
font-size: 60pt;
text-align: left;
}
.fab {
color: #00000060;
font-size: 25px;
}
.twitter, i {
color: #00000060;
font-size: 35px;
text-decoration: none;
}
a.email {
text-decoration: none;
color: #00000060;
font-size: 35px;
}
.envelope {
color: #00000060;
font-size: 5px;
text-decoration: none;
}
.poster_wrap {
width: 46in;
height: 32in;
padding: 0cm;
background-color: #ffffff;
}
span > #tab:mytable {
  font-weight: bold;
}
.section {
  padding-left: 10mm;
  padding-right: 10mm;
}
.main p {
  padding-left: 30mm;
  padding-right: 30mm;
}
.main_pic {
  width: 50%;
  display: block;
  margin-left: auto;
  margin-right: auto;
}
.orcid img {
  width: 1.5%;
}
</style>
</head>
<body>


<div class="poster_wrap">
<div class="column outer">
<br>
<div class="title section">
<h1 id="title"><em><strong>Clostridium immunis</strong></em>: <br/> A human gut commensal with therapeutic immunomodulatory effects</h1>
<br>
<h3 id="author" class="author">

Chin Yee Tan<sup> 2, 4, <a class="orcid" href="https://orcid.org/0000-0003-1104-6082"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:chin.yee.tan@duke.edu">chin.yee.tan@duke.edu</a> <br>
 Neil Surana<sup> 1, 2, 3, <a class="orcid" href="https://orcid.org/0000-0002-9544-2551"><img src="https://raw.githubusercontent.com/brentthorne/posterdown/master/images/orcid.jpg"></a></sup><br>

<a class='envelope'><i class="fas fa-envelope"></i></a> <a href="mailto:neil.surana@duke.edu">neil.surana@duke.edu</a> <br>
</h3>

<h5 id="author_extra" class="author_extra">
 </h5>

<p id="affiliation" class="affiliation">
<sup>1</sup> Department of Pediatrics, Duke University<br> <sup>2</sup> Department of Molecular Genetics and Microbiology, Duke University<br> <sup>3</sup> Department of Immunology, Duke University<br> <sup>4</sup> Duke-NUS Medical School
</p>

</div>
<div id="introduction" class="section level3">
<h3>Introduction</h3>
<ul>
<li>The modulation of our microbiome is an attractive method to abrogate disease and promote health [1]</li>
<li>The search for commensal species in humans which promote health and abrogate disease has been a high-priority but onerous venture.</li>
<li>Components of the host microbiome are known to regulate both innate adaptive immunity [2]</li>
<li>Identification of a single taxon or small consortia of taxa that promote human health will spur development of probiotic drugs for a range of diseases [3].</li>
<li>Host-microbe triangulation pioneered by Surana and Kasper [4] identified a hitherto unknown taxon of bacteria strongly associated with a protective phenotype in mouse colitis; subsequent culture and characterization led to the discovery of a new species, designated <em><strong>Clostridium immunis</strong></em>.</li>
</ul>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-1"></span>
<img src="images/CimmunisEM.jpg" alt="The first image of C.immunis captured by Transmission Electron Microscopy. \label{figurelabel}" width="30%" />
<p class="caption">
Figure 1: The first image of C.immunis captured by Transmission Electron Microscopy. 
</p>
</div>
</div>
<div id="approach" class="section level3">
<h3>Approach</h3>
<ul>
<li>Immunomodulatory effects of <em>C.immunis</em> were assessed using classical disease models with robust phenotypic readout.</li>
<li>Finding out the ‘where’ and ‘when’ of <em>C.immunis</em> colonization will educate the search for mechanistic clues.</li>
<li>Genetic manipulation of <em>C.immunis</em> is critical to identify genetically-encoded functions of its effects. <!---
### Method
- Disease models of Dextran Sodium Sulfate (DSS) colitis and Experimental Autoimmune Encephalitis (EAE) were performed as described in the literature.
- Gnotobiotic mice were orally gavaged with cultured _C.immunis_. Bacterial abundance in intestine and feces was assessed with quantitative PCR (qPCR). Intestinal immune cell profiling was done with fluorescence activated cell sorting.
--></li>
</ul>
</div>
<div id="results" class="section level3">
<h3>Results</h3>
<div id="single-oral-administration-of-c.-immunis-mitigates-clinical-severity-in-two-distinct-mouse-models-of-inflammatory-disease" class="section level4">
<h4>Single oral administration of <em>C. immunis</em> mitigates clinical severity in two distinct mouse models of inflammatory disease</h4>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-2"></span>
<img src="images/Survival.jpg" alt=" (Left) C. immunis gavage ameliorates lethality in DSS colitis and (Right) clinical severity in experimental autoimmune encephalitis. n=5 mice per group (DSS colitis); n=10 mice per group (EAE); Single dose of C.immunis preadministered before disease induction. *=p&lt;0.05 \label{figurelabel}" width="40%" /><img src="images/EAE.jpg" alt=" (Left) C. immunis gavage ameliorates lethality in DSS colitis and (Right) clinical severity in experimental autoimmune encephalitis. n=5 mice per group (DSS colitis); n=10 mice per group (EAE); Single dose of C.immunis preadministered before disease induction. *=p&lt;0.05 \label{figurelabel}" width="40%" />
<p class="caption">
Figure 2:  (Left) C. immunis gavage ameliorates lethality in DSS colitis and (Right) clinical severity in experimental autoimmune encephalitis. n=5 mice per group (DSS colitis); n=10 mice per group (EAE); Single dose of C.immunis preadministered before disease induction. *=p&lt;0.05 
</p>
</div>
</div>
<div id="c.-immunis-is-localized-to-the-cecum-and-proximal-middle-colon-presence-of-the-c.-immunis-in-the-naive-mouse-gut-after-a-single-oral-gavage-is-short-lived" class="section level4">
<h4><em>C. immunis</em> is localized to the cecum and proximal-middle colon; Presence of the <em>C. immunis</em> in the naïve mouse gut after a single oral gavage is short-lived</h4>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-3"></span>
<img src="images/Immunis%20only-HMb.jpg" alt=" (Left) C. immunis is mostly localized to the cecum and proximal-mid colon in colonized mice at steady-state, n=4 mice; (Right) C.immunis naïve C57BL/6 MMb do not retain C.immunis beyond 24hours after single oral gavage, n=9 mice; fecal pellets collected 12-hourly \label{figurelabel}" width="40%" /><img src="images/Kinetics.jpg" alt=" (Left) C. immunis is mostly localized to the cecum and proximal-mid colon in colonized mice at steady-state, n=4 mice; (Right) C.immunis naïve C57BL/6 MMb do not retain C.immunis beyond 24hours after single oral gavage, n=9 mice; fecal pellets collected 12-hourly \label{figurelabel}" width="40%" />
<p class="caption">
Figure 3:  (Left) C. immunis is mostly localized to the cecum and proximal-mid colon in colonized mice at steady-state, n=4 mice; (Right) C.immunis naïve C57BL/6 MMb do not retain C.immunis beyond 24hours after single oral gavage, n=9 mice; fecal pellets collected 12-hourly 
</p>
</div>
<!---
How do i have auto figure captioning with in-line R script? If not can I just write code to generate figure with ggplot, export and reimport with include_graphics? Also wish to facet both localization and kinetics side-by side
Edit: Decided to import PRISM made files for now for simplicity. Older commits have the code for plotting localization from raw data
--->
</div>
<div id="oral-administration-of-a-c.immunis-enrichced-culture-decreases-the-abundance-of-colonic-type-3-innate-lymphoid-cells-ilc3" class="section level4">
<h4>Oral administration of a <em>C.immunis</em> enrichced culture decreases the abundance of colonic Type 3 innate lymphoid cells (ILC3)</h4>
<div class="figure" style="text-align: center"><span id="fig:unnamed-chunk-4"></span>
<img src="images/NKp46+ILC3s.jpg" alt="Colonic type 3 innate lymphoid cells were decreased in mice gavaged with a culture enriched for C.immunis (HMb cx) compared to a culture low in C.immunis (MMb cx). **=p&lt;0.005 \label{figurelabel}" width="40%" />
<p class="caption">
Figure 4: Colonic type 3 innate lymphoid cells were decreased in mice gavaged with a culture enriched for C.immunis (HMb cx) compared to a culture low in C.immunis (MMb cx). **=p&lt;0.005 
</p>
</div>
</div>
</div>
<div id="ongoing-work" class="section level3">
<h3>Ongoing work</h3>
<div id="comparative-genomics-and-genetic-manipulation" class="section level4">
<h4>Comparative Genomics and Genetic Manipulation</h4>
<ul>
<li>I am performing whole genome comparative genomics between of closely related strains and <em>C.immunis</em> to identify candidate genes that confer protection</li>
<li>The lab is testing techniques to introduce exogenous genetic material into <em>C.immunis</em>, with the aim to performing CRISPR-mediated gene knockouts against candidate genes</li>
</ul>
</div>
<div id="detailed-mechanistic-studies-ilc3-knockout-mouse" class="section level4">
<h4>Detailed mechanistic studies: ILC3 knockout mouse</h4>
<ul>
<li>I am currently breeding ILC3 knockout mice to confirm and study the mechanism of <em>C.immunis</em> modulation of the host colonic ILC3 population</li>
</ul>
<div id="references-acknowledgements" class="section level5">
<h5>References &amp; Acknowledgements</h5>
<p><span style="font-size: 25px"> 1. Mimee, M., et al. (2016). “Microbiome therapeutics — Advances and challenges.” Advanced Drug Delivery Reviews 105: 44-54. </br> 2. Blander, J. M., et al. (2017). “Regulation of inflammation by microbiota interactions with the host.” Nature Immunology 18: 851. </br> 3. Fischbach, M. A. (2018). “Microbiome: Focus on Causation and Mechanism.” Cell 174(4): 785-790.</br> 4. Surana, N. K. and D. L. Kasper (2017). “Moving beyond microbiome-wide associations to causal microbe identification.” Nature 552: 244 </br></br> We are thankful to the Duke Gnotobiotic Core (gnotobiotic animal husbandry), the microbiome core (use of anaerobic chamber) and the Shinohara lab (advice on EAE induction). </span> <!---
[Neil's original paper]
[Review paper on search for commensal organisms with immunomodulatory potential]
[posterdown citation]
Thanks to gnotobiotic core, microbiome core and shinohara lab

---></p>
</div>
</div>
</div>

</div>
<div class="main">
<p><strong>Therapeutic Potential:</strong><br/>Oral administration of <em>Clostridium immunis</em> to mice ameliorates colitis and autoimmune encephalitis severity<br/><br/><strong>Localization and Colonization:</strong> <em>Clostridium immunis</em> localizes to the mouse cecum and proximal colon and displays short term kinetics upon single gavage<br/><br/><strong>Immunomodulatory Effects:</strong> <em>Clostridium immunis</em> modulates the intestinal type 3 innate lymphoid cell (ILC3) population</p>
<img src="images/QR.jpg" title="fig:" class="main-img-mid" />

</div>
</div>



</body>
</html>
